cmd/copyUploadEnumerator.go

```diff
@@ -25,11 +25,19 @@ func (e *copyUploadEnumerator) enumerate(cca *cookedCopyCmdArgs) error {
 	common.PanicIfErr(err)
 
 	// list the source files and directories
-	listOfFilesAndDirectories, err := filepath.Glob(cca.source)
+	// filepath.Glob incorrectly handles the long file paths indicator as a meta character, so trim it...
+	listOfFilesAndDirectories, err := filepath.Glob(common.ToShortPath(cca.source))
 	if err != nil || len(listOfFilesAndDirectories) == 0 {
 		return fmt.Errorf("cannot find source to upload")
 	}
```

Comment:
Does this mean that the code will fail here if the root path (cca.source) is itself a long path?

It looks like we can do long paths that are inside cca.source, but we can't do a long cca.source itself.  Is that right?

Reply:
Yes, filepath.Glob reads the long file path indicator's `?` as a meta character and fails to glob.